tac-operation-lifecycle/tac-operation-lifecycle-server/tests/helpers/mod.rs

```diff
@@ -0,0 +1,454 @@
+use std::sync::Arc;
+
+use blockscout_service_launcher::{test_database::TestDbGuard, test_server};
+use futures::StreamExt;
+use reqwest::Url;
+use tac_operation_lifecycle_logic::{client::settings::RpcSettings, database::TacDatabase};
+use tac_operation_lifecycle_server::Settings;
+
+pub async fn init_db(test_name: &str) -> TestDbGuard {
+    let db_name = format!("testdb_{test_name}");
```

Comment:
i think you can avoid this step and just put `test_name` into `TestDbGuard::new`. The parameter naming is kinda confusing though, I agree